content/messaging/agent-chat-api/rtm-reference/index.mdx

```diff
@@ -557,248 +593,274 @@ Custom event is an event with customizable payload.
-| `recipients`          | optional  | It can be specified when sending system messages via the [**Send Event**](#send-event) method. Possible values: `all`, `agents`.|
+### Request
+
+| Field                 | Required  | Data type | Notes                                                                                          |
+| --------------------- | --------- |---------- | ---------------------------------------------------------------------------------------------: |
+| `custom_id`           | no        | `string`  | You can give the system message a custom ID.                                                   |
+| `type`                | yes       | `string`  | `system_message`                                                                               |
+| `text`                | no        | `string`  | Text displayed to recipients                                                                   |
+| `system_message_type` | yes       | `string`  | [System message type](#available-system-messages)                                              |
+| `recipients`          | no        | `string`  | It can be specified when sending system messages via the [**Send Event**](#send-event) method. Possible values: `all`, `agents`.|
+| `text_vars`           | no        | `object`  | Variables used in the text                                                                     |
+
+### Response
+
+| Field                 | Returned   | Notes                                                                                     |
+| --------------------- | ---------- | ----------------------------------------------------------------------------------------- |
+| `id`                  | always     | Generated server-side                                                                     |
+| `custom_id`           | optionally |                                                                                           |
+| `type`                | always     |                                                                                           |
+| `created_at`          | always     | Date & time format with a resolution of microseconds, `UTC string`; generated server-side |
+| `text`                | optionally |                                                                                           |
+| `system_message_type` | always     |                                                                                           |
+| `recipients`          | optionally |                                                                                           |
+</Text>
+
+<Code>
+<CodeResponse title={'Sample System message in response'}>
+
+```json
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743",
+  "custom_id": "31-0C-1C-07-DB-16",
+  "type": "system_message",
+  "created_at": "2017-10-12T15:19:21.010200Z",
+  "text": "Hello there, %agent%!",
```

Comment:
Should I use `%` here in the example to show that it's a variable?

Reply:
I think that it can be misleading. It's an example so let's show exactly how it will look like.